js/pages/cohort-definitions/cohort-definition-manager.js

```diff
@@ -1169,6 +1169,10 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 
 		// dispose subscriptions / cleanup computed observables (non-pureComputeds)
 			dispose () {
+				let subscriptions = this.model.cohortDefinitionSubscriptions;
```

Comment:
move this dispose section over to conceptset-warnings.  The idea is to dispose in the same context that you created.